deep-prove/src/bin/worker/main.rs

```diff
@@ -29,7 +30,7 @@ type Pcs<E> = Basefold<E, BasefoldRSParams<Hasher>>;
 /// From a proof request wrapped in a [`DeepProveRequestV1`] and a [`Store`]
 /// implementation (to interact with the PPs), attempt to generate proofs for a
 /// list of inputs.
-async fn run_model_v1(model: DeepProveRequestV1, mut store: impl Store) -> Result<Vec<ProofV1>> {
+async fn run_model_v1<S: Store>(model: DeepProveRequestV1, store: &mut S) -> Result<Vec<ProofV1>> {
```

Comment:
The whole point of the store is to keep it active across multiple successive runs :)